datumaro/plugins/velodynepoints_format/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2021 cdp
```

Comment:
This license header is extraneous and has wrong license.

Reply:
Resolved